app/dashboard/tip_views.py

```diff
@@ -283,7 +283,8 @@ def send_tip_3(request):
             primary_email = to_emails['events'][0]
         else:
             print("TODO: no email found.  in the future, we should handle this case better because it's GOING to end up as a support request")
-
+    if type(primary_email) is list and len(primary_email):
```

Comment:
it might be creating a method named `get_primary_email()` that accepts `params`.. but geez, `kudos/views` and `tip_views.py` just have so much to consolidate it hurts to start